src/test/java/org/broadinstitute/hellbender/utils/reference/ReferenceUtilsUnitTest.java

```diff
@@ -52,6 +57,34 @@ public void testLoadFastaDictionaryFromStream() throws IOException {
         }
     }
 
+    @DataProvider(name = "testData_testLoadWrongFormatFastaDictionary")
+    public Object[][] testData_testLoadWrongFormatFastaDictionary(final Method testMethod) {
```

Comment:
Also, if you don't provide an explicit name for your `DataProvider`, it will default to using the method name as the `DataProvider` name.